src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationContext.cs

```diff
@@ -292,10 +292,17 @@ public Task<AuthenticationResult> AcquireTokenByDeviceCodeAsync(DeviceCodeResult
         /// <param name="userId">Identifier of the user token is requested for. This parameter can be <see cref="UserIdentifier"/>.Any.</param>
         /// <param name="extraQueryParameters">This parameter will be appended as is to the query string in the HTTP authentication request to the authority. The parameter can be null.</param>
         /// <param name="claims">Additional claims that are needed for authentication. Acquired from the AdalClaimChallengeException</param>
+        /// <param name="synchronizationContext">Optional SynchronizationContext for ensuring interactive calls are executed on the UI thread.</param>
         /// <returns>It contains Access Token and the Access Token's expiration time.</returns>
-        public async Task<AuthenticationResult> AcquireTokenAsync(string resource, string clientId, Uri redirectUri,
+        public async Task<AuthenticationResult> AcquireTokenAsync(
+            string resource, 
+            string clientId, 
+            Uri redirectUri,
             IPlatformParameters parameters,
-            UserIdentifier userId, string extraQueryParameters, string claims)
+            UserIdentifier userId, 
+            string extraQueryParameters, 
+            string claims, 
+            SynchronizationContext synchronizationContext = null)
```

Comment:
public API change (for memory)

Reply:
Yes, addition and it's for 5.0 which has breaking changes already. Any specific concerns?